### DISCUSSION TITLE
How to change the language of zitadel's Web UI?

### DISCUSSION THREAD

**Discussion by linghengqian** [zitadel/zitadel#8619] [Q&A] [answered]
- When I deploy zitadel via the `docker-compose.yml` mentioned in https://zitadel.com/docs/self-hosting/deploy/compose , apparently the default language is English. This makes sense.
- ![image](https://github.com/user-attachments/assets/733c4316-0178-4834-9dd3-67cebde11cd6)
- I observed issues like https://github.com/zitadel/zitadel/issues/5230 indicating that the zitadel web UI at http://localhost:8080/ actually has a chinese language version, and I also saw those translations at https://github.com/zitadel/zitadel/blob/v2.61.0/console/src/assets/i18n/zh.json .
- ![image](https://github.com/user-attachments/assets/e2769d01-289e-4fdb-a679-23358a622360)
- But what confuses me is that the documentation doesn't seem to mention how to change the language of the Web UI. The only i18n-related document I searched in the documentation was https://zitadel.com/docs/guides/manage/customize/texts#internationalization--i18n . But the i18n in that document seems to only be for the i18n of the login page.
- What I mean is, setting `简体中文` as the default language will not change the language used by zitadel's Web UI to Simplified Chinese.
- ![image](https://github.com/user-attachments/assets/007ac496-e2b7-4b6d-abe0-d39e072503fe)
- How to change the language of zitadel's Web UI? Or am I missing some documentation?

**Comment by hifabienne** [comment]
@peintnermax can you help here? is the default language triggered in the management console?

**Comment by peintnermax** [comment, marked as answer]
The console atm prefers the language of the user profile over the language header and falls back to english, if neither applies. The "default" setting is not considered as for now.

**Reply by linghengqian** [reply]
> The console atm prefers the language of the user profile over the language header and falls back to english, if neither applies. The "default" setting is not considered as for now.

- Thanks for the reminder, I found the way to switch to Chinese in a URL like `http://localhost:8080/ui/console/users/me?id=general`.
- ![image](https://github.com/user-attachments/assets/f6af23b0-d5ab-444b-ba72-224491d450ca)
- ![image](https://github.com/user-attachments/assets/1fa2e305-0d59-4974-9638-94b5ff3d5d99)
